### PR TITLE
General Markdown Fixes on Mint Page

### PR DESCRIPTION
**Pull request by nothingalike**
## Quickfix

I noticed there were some markdown syntax issues on the docs/minting page.

### CODE REVIEW

**Review by katomm — COMMENTED** [review]
Well spotted, my only concern is that you wiped out all of the note boxes.

**Comment by nothingalike** [comment]
@katomm were formatted that way on purpose? It just looked like more syntax errors. I can revert if that was intentional

**Comment by katomm** [comment]
The note boxes? Yes. Please see https://developers.cardano.org/docs/portal-style-guide#admonitions for details.

**Comment by nothingalike** [comment]
Gotcha. Ill see what i can do because that's definitely not rendering as the documentation is showing

File: docs/native-tokens/minting.md [hunk 1]
```diff
@@ -187,17 +187,16 @@ Policies are the defining factor under which tokens can be minted. Only those in
 We'll make a separate sub-directory in our work directory to keep everything policy-wise separated and more organized.
 For further reading, please check [the official docs](https://docs.cardano.org/native-tokens/getting-started/#tokenmintingpolicies) or the [github page about multi-signature scripts](https://github.com/input-output-hk/cardano-node/blob/c6b574229f76627a058a7e559599d2fc3f40575d/doc/reference/simple-scripts.md).
 
-"`bash
+```bash
 mkdir policy
 ```
 
-:::note
```
Comment (rdlrt) [inline docs/native-tokens/minting.md:194]: This should not be changed, as it represents "Note" admonition as per [style guide](https://developers.cardano.org/docs/portal-style-guide#admonitions)

File: docs/native-tokens/minting.md [hunk 2]
```diff
@@ -218,9 +217,7 @@ echo "  \"type\": \"sig\"" >> policy/policy.script
 echo "}" >> policy/policy.script
 ```
 
-:::note
```
Comment (rdlrt) [inline docs/native-tokens/minting.md:221]: Same as previous comment

File: docs/native-tokens/minting.md [hunk 3]
```diff
@@ -440,10 +435,9 @@ A few things worth pointing out:
 3. Our own address, therefore, needs to receive our funds, subtracted by the transaction fee as well as the minimum of 1 ada we need to send to the other address and
 4. all of the tokens the txhash currently holds, subtracted by the tokens we send.
 
-:::note
-Depending on the size and amount of native assets you are going to send it might be possible to send more than the minimum requirement of only 1 ada. For this guide, we will be sending 10 ada to be on the safe side.
```
Comment (rdlrt) [inline docs/native-tokens/minting.md:444]: Same as previous comment

File: docs/native-tokens/minting.md [hunk 4]
```diff
@@ -507,13 +501,11 @@ If you've followed this guide up to this point, you should be familiar with the
 
 Set everything up and check our address:
 
-"`bash
+```bash
 cardano-cli query utxo --address $address --$testnet
 ```
 
-:::note
```
Comment (rdlrt) [inline docs/native-tokens/minting.md:514]: Same as previous comment

File: docs/native-tokens/minting.md [hunk 5]
```diff
@@ -538,12 +530,11 @@ cardano-cli transaction build-raw \
  --out-file burning.raw
  ```
 
-:::note
```
Comment (rdlrt) [inline docs/native-tokens/minting.md:541]: Same as previous comment

**Review by rdlrt — CHANGES_REQUESTED** [review]
Thanks for the contribution. The admonitions were correct for docusaurus as per style guide. The syntax highlighting for code section was indeed wrong, but would be good to fix other instances on the minting page too

**Comment by nothingalike** [comment]
@katomm @rdlrt I reverted my changes. Any idea why its not rendering as show in reference?

**Comment by nothingalike** [comment]
Ok found more bash syntax issues, but i see why i thought things were not rendering correctly. my apologies. everything should be good now

**Review by rdlrt — APPROVED** [review]

**Review by katomm — APPROVED** [review]
Thank you @nothingalike

**Comment by katomm** [comment]
> but i see why i thought things were not rendering correctly. 

I am curious what the problem was @nothingalike.

**Comment by nothingalike** [comment]
@katomm  It was the additional code syntax issues i found.

**Review by tweakch — APPROVED** [review]

**Pull request merged**